### DISCUSSION TITLE
Background color in <body> tag

### DISCUSSION THREAD

**Discussion by VijayShankar1** [edrlab/thorium-reader#1817] [General]
If we provide color to the background in `<body>` tag using external stylesheet. The **Scrollable view** in Thorium, shows the Landing page view only having background color, rest of the below contents don't have the background colors. We have attached the screenshot for your reference (Opener_view.jpg and Bottom_View.jpg).

If we set the display option as **Paginated View**, the background color is viewable correctly (screenshot attached Paginated_View.jpg).

Kindly fix this as bug.

![Opener_view](https://user-images.githubusercontent.com/105100449/189039420-18e28729-3b93-4f66-bae8-02bb2e7421f3.jpg)
![Bottom_View](https://user-images.githubusercontent.com/105100449/189039432-ea589d8e-15db-4adb-b584-ae31fa22f24e.jpg)
![Paginated_View](https://user-images.githubusercontent.com/105100449/189039437-df5d0573-eabd-4596-8335-9b5463c591be.jpg)

**Comment by danielweck** [comment]
This is a bug, so I created an issue: https://github.com/edrlab/thorium-reader/issues/1824
(please continue the discussion there, not here :)
